06-building-a-program-start-to-finish/solving-the-problem/output/app/class/invertedIndexClass.js

```diff
@@ -0,0 +1,202 @@
+/**
+ * Inverted Index class
+ * @class
+ */
+class InvertedIndex {
+
+  /**
+   * class constructor
+   * @constructor
+  */
+  constructor() {
+    this.indices = {};
+    this.indexedFiles = {};
+  }
+
+  /**
+   * function to read uploaded files
+   * @function
+   * @param {object} file
+   * @returns promise
```

Comment:
What type are you returning. Eslint on this line error - (Missing JSDoc return type)